extra/resources/controld

```diff
@@ -228,7 +228,14 @@ controld_monitor() {
     killall -0 ${OCF_RESKEY_daemon} >/dev/null 2>&1 ; rc=$?
 
     case $rc in
-      0) rc=$OCF_SUCCESS;;
+      0) dlm_tool ls | grep -q "wait fencing"
```

Comment:
Parsing output meant for humans can be brittle (though unfortunately often the only way). Is there any way to get the info from machine-readable output (hopefully more stable over the long term) such as XML or an exit code?

It might be a good idea to redirect stderr to /dev/null.

It also might be a good idea to do this in two steps instead of a pipe, so you can check the exit code of dlm_tool. If dlm_tool returns an error, it might not say "wait fencing", but we might want to return an error anyway to be on the safe side.